client/checkout/classic/upe-deferred-intent-creation/stripe-checkout.js

```diff
@@ -64,12 +64,13 @@ function blockUI( jQueryForm ) {
  *
  * @param {Object} elements The Stripe elements object to be validated.
  * @param {Object} jQueryForm The jQuery object for the form being validated.
+ * @return {Promise} Promise for the checkout submission.
  */
 function validateElements( elements, jQueryForm ) {
-	elements.submit().then( ( result ) => {
+	return elements.submit().then( ( result ) => {
 		if ( result.error ) {
 			jQueryForm.removeClass( 'processing' ).unblock();
```

Comment:
```suggestion
```
Think we can remove this too, now that it'll be handled in the other `catch` block. 👍

Reply:
I missed that! Thanks for catching it. Removed in https://github.com/Automattic/woocommerce-payments/pull/6262/commits/6151283ef672c351509da085b0551b17e485a6cb